edx_notifications/server/api/consumer.py

```diff
@@ -211,6 +212,25 @@ def post(self, request, msg_id):
         return Response({}, status.HTTP_200_OK)
 
 
+class MarkNotifications(AuthenticatedAPIView):
```

Comment:
As this is the first POST method on this API, we'll have to investigate how to ingrate CSRF tokens to prevent cross-site scripting. I'll do some researching.